src/main/msp/msp.c

```diff
@@ -1523,6 +1523,12 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0);
 #endif
 
+#if defined(USE_ITERM_RELAX)
```

Comment:
Please add the '// Added in MSP API 1.42' comment here as well.

Reply:
Added the comment